jmclient/jmclient/configure.py

```diff
@@ -329,6 +329,13 @@ def jm_single():
 # for payjoins to hidden service endpoints, the socks5 configuration:
 onion_socks5_host = localhost
 onion_socks5_port = 9050
+
+# for payjoin hidden service creation, the tor control configuration:
```

Comment:
There is the same mistake in a comment above, but Tor documentation nowadays call it "onion service", not "hidden service".

Reply:
Fixed in fde39a7, I didn't fix the above comment because it is out-of-scope for this PR.